src/custom/utils/trade.ts

```diff
@@ -25,14 +26,14 @@ export interface PostOrderParams {
 }
 
 function _getSummary(params: PostOrderParams): string {
-  const { inputAmount, outputAmount, account, recipient, recipientAddressOrName } = params
+  const { inputAmount, adjustedOutputAmount, account, recipient, recipientAddressOrName } = params
 
   const inputSymbol = inputAmount.currency.symbol
-  const outputSymbol = outputAmount.currency.symbol
+  const outputSymbol = adjustedOutputAmount.currency.symbol
   const inputAmountValue = inputAmount.toSignificant(SHORTEST_PRECISION)
-  const outputAmountValue = outputAmount.toSignificant(SHORTEST_PRECISION)
+  const outputAmountValue = adjustedOutputAmount.toSignificant(SHORTEST_PRECISION)
 
-  const base = `Swap ${inputAmountValue} ${inputSymbol} for ${outputAmountValue} ${outputSymbol}`
+  const base = `Swap ${inputAmountValue} ${inputSymbol} for at least ${outputAmountValue} ${outputSymbol}`
```

Comment:
Happy to see this change, why is not mentioned in the description?

Reply:
:shrug: it as going to get done either way thought i was explicity enough with my title i guess. anyways, it's here now!